.circleci/config.yml

```diff
@@ -0,0 +1,235 @@
+version: 2
+references:
+  build_containers_config: &build_container_config
+    working_directory: ~/repo
+    docker:
+      - image: circleci/ruby:2.5.3-node-browsers
+        environment:
+          BUNDLE_JOBS: 3
+          BUNDLE_PATH: vendor/bundle
+          BUNDLE_RETRY: 3
+          CBO_BASE_DATABASE_DATABASE: cccd_test
+          PDFTK_PATH: /usr/bin/pdftk
+          PGHOST: 127.0.0.1
+          PGUSER: postgres
+          RAILS_ENV: test
+          TZ: Europe/London
+      - image: circleci/postgres:9.6-alpine
+        environment:
+          POSTGRES_USER: postgres
+          POSTGRES_DB: cccd_test
+          POSTGRES_PASSWORD: ""
+
+  install_phantom_js: &install_phantom_js
```

Comment:
do we need this now? have removed this from feature tests in favour of headless chrome since you started working on this

Reply:
**NOTE a blocker** As discussed you, apparently, can append `-browsers-legacy` to the docker image tag name to include PhantomJS

https://circleci.com/docs/2.0/browser-testing/

**Not a blocker**
An to switch jasmine over to using headless chrome i found this started for 10
https://github.com/jasmine/jasmine-gem/issues/293

Reply:
the `node-browsers-legacy` fix worked 💥